Hunted Mobile/Hunted Mobile/Hunted Mobile/ViewModel/MapViewModel.cs

```diff
@@ -22,11 +22,10 @@
 using Xamarin.Forms;
 using System.Timers;
 using Newtonsoft.Json.Linq;
-using System.Windows.Input;
 
 namespace Hunted_Mobile.ViewModel {
     public class MapViewModel : BaseViewModel {
-        private MapView _mapView;
+        private MapView _mapView = null;
```

Comment:
Dit is standaard al `null` als het niet wordt gezet, dus een standaard waarde is heir niet relevant.